src/calculateRentalCost.js

```diff
@@ -3,8 +3,26 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const totalRentPrice = days * rentCost;
+  const longTermRent = 7;
+  const shortTermRent = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  if (days < shortTermRent) {
+    return totalRentPrice;
+  }
+
+  if (days >= shortTermRent && days <= 6) {
+    return totalRentPrice - shortTermDiscount;
+  }
+
+  if (days >= longTermRent) {
+    return totalRentPrice - longTermDiscount;
```

Comment:
Since the previous conditions return early, there's no need for the `if` statement here. You can simply return `totalRentPrice - longTermDiscount;` as the default case after the previous `if` statement. This follows the checklist item 5, which advises against adding `else` or additional `if` after an `if` with `return`.